main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -82,13 +83,21 @@ public String getCreateSQL() {
       
                 
                 if (name != null) {
-                    if(trimColNames) {
+                	if(trimColNames) {
                         String trimmedCol = name.replaceAll("\\s", "");
-                        createSB.append( trimmedCol + " ");
-                    }else{
-                        createSB.append(name + " ");
-                    }
-                   
+                        if(dbType.equals("PostgreSQL")) {
+                    	       createSB.append("\"" +trimmedCol.replaceAll("'","''")+ "\" ");
+                    	   }else {
+                    		   createSB.append("`"+trimmedCol+"`" + " ");
+                    	   }
+                     }else{
+                     	if(dbType.equals("PostgreSQL")) {
+                     	    createSB.append("\"" +name.replaceAll("'","''")+ "\" ");
+                     	}else {
+                     		createSB.append("`"+name+"` ");	
+                     	}
+                     }
```

Comment:
Indentation looks off here. Please make sure all indents are 4 spaces.

Reply:
Yes sir I will work on indentation and try to encapsulate repeating code in a method.

Reply:
I have declared function like this:
![image](https://user-images.githubusercontent.com/59737567/109871637-65813200-7c91-11eb-8561-d38323415a8a.png)
does this look fine now?
![image](https://user-images.githubusercontent.com/59737567/109871793-91041c80-7c91-11eb-9be1-ac70123a11d8.png)